test/Unipool.js

```diff
@@ -40,6 +41,10 @@ require('chai').use(function (chai, utils) {
     });
 });
 
+const assertBalance = async function (promise, expectedValue) {
+    expect(await promise).to.be.bignumber.almostEqualDiv1e18(wei(expectedValue));
```

Comment:
Would it make sense to keep an strictly equal version like in the original tests?